_episodes/07-Practical Considerations:Reporting-results.md

```diff
@@ -69,9 +69,9 @@ In addition to predictions (e.g. "disease" or "not disease" labels), some ML alg
 <img src="../fig/ROC.png" alt="drawing" width="350"/>
 </p> |
 | :--: | 
-| Receiver Operator Characteristic (ROC) Curve. The y-axis shows True Positive Rate (TPR), also called sensitivity or recall. The x-axis is False Positive Rate (FPR) or 1 - specificity. The ROC curve plots the FPR against TPR for various thresholds, that is, the points along the blue curve.|
+| Receiver Operator Characteristic (ROC) Curve. The y-axis shows True Positive Rate (TPR), also called sensitivity or recall. The x-axis is False Positive Rate (FPR) or 1 - specificity. The ROC curve plots the FPR against TPR for various thresholds, i.e., the points along the blue curve.|
 
-The orange dashed line shows the ROC curve of a purely random classifier. For such classifiers, `sensitivity = 1 - specificity` (or TPR = FPR) at all points. The blue curve shows the TPR vs FPR at all threshold values, and a the further the curve is shifted toward the top left of the graph (TPR=1,FPR=0). We can also calculate the *Area Under the Curve (AUC)* that allows us to see the the total aggregated measure of TPR and FPR over all thresholds and is often reported along with accuracy.
+The orange dashed line shows the ROC curve of a purely random classifier. For such classifiers, `sensitivity = 1 - specificity` (or TPR = FPR) at all points. The blue curve shows the TPR vs FPR at all threshold values, and the further the curve is shifted toward the top left of the graph (TPR=1, FPR=0). We can also calculate the *Area Under the Curve (AUC)* that allows us to see the total aggregated measure of TPR and FPR over all thresholds and is often reported along with accuracy.
```

Comment:
> the further the curve is shifted toward the top left of the graph (TPR=1, FPR=0).

better?

-> Maybe we can reformulate this sentence, e.g., The ROC curve of a good classifier is closer to the top-left corner. its